packages/dds/tree/api-report/tree.legacy.public.api.md

```diff
@@ -422,23 +422,29 @@ export class SchemaFactory<out TScope extends string | undefined = string | unde
     readonly requiredRecursive: <const T extends ImplicitAllowedTypesUnsafe, const TCustomMetadata = unknown>(t: T, props?: Omit<FieldProps<TCustomMetadata>, "defaultProvider"> | undefined) => FieldSchemaUnsafe<FieldKind.Required, T, TCustomMetadata>;
     static readonly requiredRecursive: <const T extends ImplicitAllowedTypesUnsafe, const TCustomMetadata = unknown>(t: T, props?: Omit<FieldProps<TCustomMetadata>, "defaultProvider"> | undefined) => FieldSchemaUnsafe<FieldKind.Required, T, TCustomMetadata>;
     readonly scope: TScope;
-    readonly string: LeafSchema_2<"string", string>;
-    static readonly string: LeafSchema_2<"string", string>;
-}
-
-// @public @sealed
-export const schemaStatics: {
-    readonly string: LeafSchema_2<"string", string>;
-    readonly number: LeafSchema_2<"number", number>;
-    readonly boolean: LeafSchema_2<"boolean", boolean>;
-    readonly null: LeafSchema_2<"null", null>;
-    readonly handle: LeafSchema_2<"handle", IFluidHandle<unknown>>;
-    readonly leaves: readonly [LeafSchema_2<"string", string>, LeafSchema_2<"number", number>, LeafSchema_2<"boolean", boolean>, LeafSchema_2<"null", null>, LeafSchema_2<"handle", IFluidHandle<unknown>>];
+    readonly string: LeafSchema<"string", string>;
+    static readonly string: LeafSchema<"string", string>;
+}
+
+// @public @sealed
+export interface SchemaStatics {
```

Comment:
Note for any other users: this export is tagged as `@system`, and we therefore are allowed to make this breaking change.

Reply:
it was also only type exported, so no one could actually use it at runtime.